.github/workflows/removed-slices.yaml

```diff
@@ -0,0 +1,48 @@
+name: "Removed slices"
+
+on:
+  push:
+    paths:
+      - "slices/**"
+  pull_request:
```

Comment:
Why do you need this ? The branch is already using the workflow call on every PR. Won't it cause duplicated checks? 

In your demo we actually see the job running twice: https://github.com/rebornplusplus/chisel-releases/pull/17/checks, although one is triggered by the push event.

Reply:
Done in bd68502.